team-telnyx/webrtc
# feat(ENGDESK-27107): Upgrade WebRTC library to support React 18

- Upgrade WebRTC library to support React 18

## 📝 To Do


## ✋ Manual testing

1. Navigate to  `packages/react-client`
2. Run yarn build
3. it should build without issues.
4. run yarn link
5. go to https://github.com/team-telnyx/webrtc-examples/pull/59
6. clone and install dependencies
7. run yarn link "@telnyx/react-client"
8. run the project and make a call

## 🦊 Browser testing

### Desktop
